Story/Episode 6/st_062080.csv

```diff
@@ -0,0 +1,21 @@
+st_062080_0001#0,"""A.P.242/2/26/16:10"""
+st_062080_0010#0,"""Is that all? Surely the Guardian who displayed such fearsome bravery in Omega should be capable of more."""
+st_062080_0020#0,"""...So we really are enemies."""
+st_062080_0030#0,"""Why did you do what you did in Omega?"""
+st_062080_0040#0,"""Whatever do you mean. Have I not stated that this was all for my liege?"""
+st_062080_0050#0,"""I have already told you. It is all for the sake of my liege."""
+st_062080_0060#0,"""Everything I have done, has been for the sole purpose of returning my liege to this world."""
+st_062080_0070#0,"""Concealing myself in Omega, playing the role of dutiful knight, fighting alongside you, saving the world... Welcoming in my liege."""
+st_062080_0080#0,"""...And finally, fulfilling her wish to destroy this world."""
+st_062080_0090#0,"""That is her purpose... That is the purpose of Varuna, God of Fidelity."""
+st_062080_0100#0,"""God of Fidelity..."""
+st_062080_0110#0,"""Mithra and I owe our existence to our liege, and we are likewise required of her."""
```

Comment:
We're calling her Mitra to make sense of the Vedic lore between Varuna, Mitra and Shiva.